File: SimpleClient/SimpleClientWordProcessor/SimpleClientWordProcessor/poco/MostUsedWord.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SimpleClientWordProcessor
{
    [Serializable]
    public class MostUsedWord
    {
        public string word;
        public int count;


        public MostUsedWord(string word, int count)
        {
            this.word = word;
            this.count = count;
        }

    }
}

File: ServiceCode/FileAnalyzer/src/com/clientfileoperation/MostUsedWord.java
package com.clientfileoperation;

import java.io.Serializable;

/**
 * Created by Caner on 23.01.2017.
 */
public class MostUsedWord implements Serializable {
    public String word;
    public int count;


    public MostUsedWord(String word, int count)
    {
        this.word = word;
        this.count = count;
    }
}
